### PR TITLE
ImgRecs: make bottomsheet draggable

### PR DESCRIPTION
**Pull request by cooltey**
https://phabricator.wikimedia.org/T280151

### CODE REVIEW

**Comment by sharvaniharan** [comment]
It works really well! at first glance! We only need to adjust the peek height as some fraction of the screen height and we are all set! Great work @cooltey  !! 👏🏼 👏🏼 Will review code in a bit!

**Comment by sharvaniharan** [comment]
Checking this..

**Comment by sharvaniharan** [comment]
@cooltey Looks like the peek height is not optimized for small screen?
peek

We should be able to see the article, no?

**Review by sharvaniharan — CHANGES_REQUESTED** [review]
Needs to be optimized for small screen.

**Comment by cooltey** [comment]
> 
> 
> Needs to be optimized for small screen.

Hi @sharvaniharan 
I changed the default visibility of bottom sheet to collapsed with the new peek height, and I'll leave comment to Robin about this change.

**Comment by sharvaniharan** [comment]
ok @cooltey Will merge it then..

**Comment by cooltey** [comment]
Thanks @sharvaniharan, please see the comment here https://phabricator.wikimedia.org/T280151#7022461 and merge it if it looks good to you 🙏

File: app/src/main/java/org/wikipedia/suggestededits/ImageRecsFragment.kt [hunk 1]
```diff
@@ -269,7 +276,13 @@ class ImageRecsFragment : SuggestedEditsItemFragment(), ImageRecsDialog.Callback
 
                     binding.articleScrollSpacer.post {
                         if (isAdded) {
-                            binding.articleScrollSpacer.layoutParams = LinearLayout.LayoutParams(LinearLayout.LayoutParams.MATCH_PARENT, binding.imageSuggestionContainer.height)
+                            binding.articleScrollSpacer.layoutParams = LinearLayout.LayoutParams(LinearLayout.LayoutParams.MATCH_PARENT, bottomSheetBehavior.peekHeight)
+                            // Collapse bottom sheet if the article title is not visible when loaded
+                            binding.suggestedEditsItemRootView.doViewsOverlap(binding.articleTitle, binding.bottomSheetCoordinatorLayout).run {
```
Comment (sharvaniharan) [inline app/src/main/java/org/wikipedia/suggestededits/ImageRecsFragment.kt:281]: 👍

**Pull request merged**